### DISCUSSION TITLE
Edit Calendar

### DISCUSSION THREAD

**Discussion by kamiro16** [YetiForceCompany/YetiForceCRM#15280] [How to] [answered]
Hello, 

![calendar](https://user-images.githubusercontent.com/65873917/136375753-98e969c1-2c70-4ac0-8759-1317a803a66f.png)

How can I add a description to the calendar view?
Which TPL file is responsible for this look?

Thanks for any help.

**Comment by vovpff** [comment]
I think you need modify calendar view template. This is not option. You need programmer

**Comment by kamiro16** [comment]
@vovpff 

Thanks you for answer. 
Unfortunately I can't find TPL file who is responsible for it.

**Reply by vovpff** [reply]
Check around here /layouts/.../modules/Calendar/

**Comment by kamiro16** [comment]
@vovpff 

I searched through all the files, code and found nothing to match.

**Reply by vovpff** [reply]
I'm far away from my server and can't help you quick

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
the easiest way to add new field to the title or modify the Js rendering view and the action of fetching data

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/blob/280ef411eb6b35313449fb1fd697984a25d00cd4/modules/Calendar/models/Calendar.php#L127

**Reply by kamiro16** [reply]
@mariuszkrzaczkowski 
Thank you, 

Unfortunately, the system only outputs the record ID, not the label.
It is process field. 
But all works good. 

![obraz](https://user-images.githubusercontent.com/65873917/136531068-297bb628-57c3-4a9b-a61f-2a27eee13eb9.png)

**Reply by vovpff** [reply]
Try getDistlaValue() instead get()

**Reply by mariuszkrzaczkowski** [reply]
there is no object but data from DB, better to use \App\Record::getLabel($id)

**Comment by kamiro16** [comment]
@mariuszkrzaczkowski 
Thank you. I will try.
